Dockerfile

```diff
@@ -42,3 +42,7 @@ RUN rm -f /etc/cos/config
 # Download rancherd
 ARG RANCHERD_VERSION=v0.0.1-alpha14
 RUN curl -o /usr/bin/rancherd -sfL "https://github.com/rancher/rancherd/releases/download/${RANCHERD_VERSION}/rancherd-amd64" && chmod 0755 /usr/bin/rancherd
+
+# Download virtctl
+ARG VIRTCTL_VERSION=v0.58.1
```

Comment:
Prefer to use https://github.com/kubevirt/kubevirt/releases/tag/v0.55.2 as this will be more compatible with our embedded v0.54.x version and also contains the required CVE fixes.